ccd-definition/CaseEventToFields.json

```diff
@@ -3,8 +3,20 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "Shared_Storage_DRAFTType",
     "CaseEventID": "openCase",
-    "CaseFieldID": "caseName",
+    "CaseFieldID": "caseLocalAuthority",
     "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "caseName=\"null\""
```

Comment:
:( @swalker125 , have you got any advise from CCD on this?

Reply:
@geek-soft I did not. I don't think it is possible in any other way. I did get a reply saying please let me know if you do discover a way though 😂